cmd/flowserver/flow.go

```diff
@@ -38,6 +38,7 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 type startRequest struct {
 	Environment *json.RawMessage       `json:"environment"`
 	Flows       json.RawMessage        `json:"flows"    validate:"required"`
+	Channels    []json.RawMessage      `json:"channels,omit_empty"`
```

Comment:
No need to change this now, but food for thought, should we just change to pass in a `SessionEnvironment` here instead of breaking out all the things we use to build said `SessionEnvironment`.. seems we are breaking this apart and then basically having two formats for what is a `SessionEnvironment`